.github/workflows/runner.yml

```
@@ -0,0 +1,34 @@
+name: 🔗 Tiered Pipeline Runner
+on: [push, pull_request]
+
+concurrency:
+  group: ci-${{github.actor}}-${{github.head_ref || github.run_number}}-${{github.ref}}-runner
+  cancel-in-progress: true
+
+jobs:
+  static-checks:
```

```suggestion
  static-checks:
    name: 📊 Static
```

See https://github.com/godotengine/godot/pull/75338#issuecomment-1485488463.